private/define-scoped-transformers.scrbl

```diff
@@ -9,6 +9,6 @@ and friends.
 @defform[(define-syntax-with-scoped-pre-transformers id
            ([stx-lens pre-transformer] ...)
            transformer-expr)]{
- Binds @racket[id] as a syntax transformer equivalent to
```

Comment:
This change is a little ambiguous. The original wording was meant to state that the *syntax transformer* was equivalent to the given expression, but separating them into two sentences makes it sound like the entire use of `define-syntax-with-scoped-pre-transformers` is equivalent to that expression.